plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/PropertyNameResolver.kt

```diff
@@ -31,4 +37,14 @@ class PropertyNameResolver(private val scopeDepth: Int, val parent: PropertyName
     }
 
     fun innerScope(scopeDepth: Int) = PropertyNameResolver(scopeDepth, this)
+
+    fun addLoopIdentifier(labelName: String, index: Int) = PropertyNameResolver(scopeDepth, parent, LoopIdentifier(labelName, index))
```

Comment:
The name of this function is a bit confusing since it actually replaces the loop identifier.  Is there actually any benefit to calling this the *loop* identifier?  Would it help to somewhere communicate that this is just the name of this scope?

Reply:
Are you suggesting that the name of the data class `LoopIdentifier` could be more general, or are you also proposing that the function `addLoopIdentifier` should be made more general?

Reply:
I mean, it's a bit confusing that if you try to use two loop identifiers in one scope, you end up dropping the outer one.  But maybe this is fine since every loop is its own scope.  I just wish we had a nicer way to express that.